RELEASE_STEPS.md

```diff
@@ -40,7 +40,7 @@ Do changes for a release:
 - Add an entry to ChangeLog, remember to change the day of the week in the release date
   - For major releases mention API and ABI compatibility with the previous release
 - Update project's version in a few places:
-  - Set the new $VERSION in `project` function in the top-level `CMakeLists.txt`
+  - For major and minor releases: `UMF_VERSION_CURRENT` in `include/umf/base.h` (the API version)
```

Comment:
What about `UMF_VERSION_CURRENT` ? Shouldn't it be equal to `UMF_VERSION` ?
I think that `umfGetCurrentVersion()` should return the new `UMF_VERSION`.

Reply:
I see. OK. Thanks for the explanation!